apps/builddao/widget/app.jsx

```diff
@@ -40,6 +40,170 @@ const Root = styled.div`
+        format("embedded-opentype"),
+      url("https://cdn.jsdelivr.net/gh/webfontworld/Poppins/Poppins-SemiBold.woff2")
+        format("woff2"),
+      url("https://cdn.jsdelivr.net/gh/webfontworld/Poppins/Poppins-SemiBold.woff")
+        format("woff"),
+      url("https://cdn.jsdelivr.net/gh/webfontworld/Poppins/Poppins-SemiBold.ttf")
+        format("truetype");
+    font-display: swap;
+  }
+  @font-face {
+    font-family: "Poppins";
+    font-weight: 700;
+    font-style: normal;
+    src: url("https://cdn.jsdelivr.net/gh/webfontworld/Poppins/Poppins-Bold.eot");
+    src:
+      url("https://cdn.jsdelivr.net/gh/webfontworld/Poppins/Poppins-Bold.eot?#iefix")
+        format("embedded-opentype"),
+      url("https://cdn.jsdelivr.net/gh/webfontworld/Poppins/Poppins-Bold.woff2")
+        format("woff2"),
+      url("https://cdn.jsdelivr.net/gh/webfontworld/Poppins/Poppins-Bold.woff")
+        format("woff"),
+      url("https://cdn.jsdelivr.net/gh/webfontworld/Poppins/Poppins-Bold.ttf")
+        format("truetype");
+    font-display: swap;
+  }
+  @font-face {
+    font-family: "Poppins";
+    font-weight: 800;
+    font-style: normal;
+    src: url("https://cdn.jsdelivr.net/gh/webfontworld/Poppins/Poppins-ExtraBold.eot");
+    src:
+      url("https://cdn.jsdelivr.net/gh/webfontworld/Poppins/Poppins-ExtraBold.eot?#iefix")
+        format("embedded-opentype"),
+      url("https://cdn.jsdelivr.net/gh/webfontworld/Poppins/Poppins-ExtraBold.woff2")
+        format("woff2"),
+      url("https://cdn.jsdelivr.net/gh/webfontworld/Poppins/Poppins-ExtraBold.woff")
+        format("woff"),
+      url("https://cdn.jsdelivr.net/gh/webfontworld/Poppins/Poppins-ExtraBold.ttf")
+        format("truetype");
+    font-display: swap;
+  }
+  @font-face {
+    font-family: "Poppins";
+    font-weight: 900;
+    font-style: normal;
+    src: url("https://cdn.jsdelivr.net/gh/webfontworld/Poppins/Poppins-Black.eot");
+    src:
+      url("https://cdn.jsdelivr.net/gh/webfontworld/Poppins/Poppins-Black.eot?#iefix")
+        format("embedded-opentype"),
+      url("https://cdn.jsdelivr.net/gh/webfontworld/Poppins/Poppins-Black.woff2")
+        format("woff2"),
+      url("https://cdn.jsdelivr.net/gh/webfontworld/Poppins/Poppins-Black.woff")
+        format("woff"),
+      url("https://cdn.jsdelivr.net/gh/webfontworld/Poppins/Poppins-Black.ttf")
+        format("truetype");
+    font-display: swap;
+  }
+
+  /* Inter Font */
+  @font-face {
+    font-family: InterVariable;
+    font-style: normal;
+    font-weight: 100 900;
+    font-display: swap;
+    src: url("https://rsms.me/inter/font-files/InterVariable.woff2?v=4.0")
+      format("woff2");
+  }
+  @font-face {
+    font-family: InterVariable;
+    font-style: italic;
+    font-weight: 100 900;
+    font-display: swap;
+    src: url("https://rsms.me/inter/font-files/InterVariable-Italic.woff2?v=4.0")
+      format("woff2");
+  }
```

Comment:
I see fontFamily in some of the children -- any way we can just have that defined here for p, h1, h2, or whatever are using it

Reply:
it depends on where it is being used. But I have globally used Inter font so it looks really nice.